lib/rules/no-unregistered-components.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @fileoverview Report used components that are not registered
+ * @author Jesús Ángel González Novez
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const utils = require('eslint-plugin-vue/lib/utils')
+const casing = require('eslint-plugin-vue/lib/utils/casing')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'disallow using components that are not registered',
+      categories: ['essential'],
```

Comment:
The user must configure the globally registered components. So the category cannot be `'essential'`.
Please set `undefined`.

Reply:
You are right, I've changed it :+1: